backend/src/packages/auth/auth.service.ts

```diff
@@ -181,9 +183,14 @@ class AuthService {
 
     const updatedUser = await this.generateAccessTokenAndUpdateUser(user.id);
 
+    const userBusiness = await this.businessService.findByOwnerId(
+      updatedUser.id,
+    );
+
```

Comment:
maybe it's better to search for business details only if user group is business?